backend/tomato/host/__init__.py

```diff
@@ -100,6 +100,7 @@ class Host(DumpSource, Entity, BaseDocument):
 	dumpLastFetch = FloatField(db_field='dump_last_fetch')
 	templates = ListField(ReferenceField(Template, reverse_delete_rule=PULL))
 	hostNetworks = ListField(db_field='host_networks')
+	synchronized = True
```

Comment:
This does not work. This variable is not stored in the database and will be lost when the object is fetched by a different operation.